proxies/shared/partials/Partial.Flows.AddPayloadToPing.xml

```diff
@@ -2,9 +2,13 @@
     <Description/>
     <Request/>
     <Response>
+        <Step>
+          <Name>RaiseFault.405NotAllowed</Name>
+          <Condition>(request.verb != "GET") and (request.verb != "HEAD")</Condition>
+        </Step>
         <Step>
             <Name>AssignMessage.AddPayloadToPing</Name>
         </Step>
     </Response>
-    <Condition>(proxy.pathsuffix MatchesPath "/_ping") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
```

Comment:
We shouldn't be modifying this condition for the `/_ping` endpoint, it needs to only work for `GET` or `HEAD` requests.